doc/spec_exceptionHandling.dox

```diff
@@ -211,8 +214,8 @@ Note: This section defines the internal interface on a low level. Helper functio
 
 - 2.8 In doPostRead()/doPostWrite() also boost::numeric::bad_numeric_cast exceptions are caught.
   - 2.8.1 The exception is *not* reported to the DeviceModule (in contrast to \ref c_2_7_1 "2.7.1")
-  - 2.8.1 In doPostWrite(), the exception is just caught, but no further action is required. The transfer itself has been skipped because the original exception occured in doPreWrite(), and dataLost ist already true.
-  - 2.8.2 In doPostRead() the DataValidity returned by the accessor is overridden to faulty until the next successful read operation, and currentVersion is taken from the successful transfer.
+  - 2.8.2 In doPostWrite(), the exception is just caught, but no further action is required. The transfer itself has been skipped because the original exception occured in doPreWrite(), and dataLost ist already true. (cf. \ref b_6 "B.6")
```

Comment:
This (incl. B.6) might be wrong according to TransferElement Spec B.7.2 : "If it (writeTransfer) returns true, **previous** data was rejected in the process of the transfer. It is always guaranteed that the **data of the current transfer is not lost**. We have the exception to transport this information.
However, the TE spec does not say that the return value of write() is the same as of writeTransfer().
I think the correct behaviour would be to flag the data as invalid, because it is the old value. The new one could not be processed. But what to return on write()? False would also be wrong.

Reply:
B.6.2 it formally wrong for a very simple reason: If an exception is thrown, there is no return value.

Reply:
Well, we are catching the exception and now have to come up with a valid return value. That's the problem.